WordPress/Sniff.php

```diff
@@ -446,6 +446,218 @@ protected function is_sanitized( $stackPtr ) {
 		// Check if this is a sanitizing function.
 		return in_array( $functionName, WordPress_Sniffs_XSS_EscapeOutputSniff::$sanitizingFunctions );
 	}
+
+	/**
+	 * Get the index key of an array variable.
+	 *
+	 * E.g., "bar" in $foo['bar'].
+	 *
+	 * @since 0.5.0
+	 *
+	 * @param int $stackPtr The index of the token in the stack.
+	 *
+	 * @return string|false The array index key whose value is being accessed.
+	 */
+	protected function get_array_access_key( $stackPtr ) {
+
+		// Find the next non-empty token.
+		$open_bracket = $this->phpcsFile->findNext(
+			PHP_CodeSniffer_Tokens::$emptyTokens
+			, $stackPtr + 1
+			, null
+			, true
```

Comment:
Yeah, it's a personal preference that's become a habit. I've found that it really does reduce confusion between function calls and arrays (since arrays will always have the commas in back). I can (try to) refrain from it if you'd like. :smile: